Dockerfile_mbtci_template

```diff
@@ -135,39 +147,74 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
+    && mkdir -p ${GOPATH} \
+    && ln -s ${GO_HOME} ${GOPATH} \
+    && chmod -R 777 ${GOPATH} \
     && rm -rf "$GNUPGHOME" go${GO_VERSION}.${GOOS}-${ARCH}.tar.gz go${GO_VERSION}.${GOOS}-${ARCH}.tar.gz.asc \
     && apt-get remove --purge --autoremove -y ca-certificates curl gnupg dirmngr \
     # smoke test
     && echo "go ${GO_VERSION} install smoke test!" \
     && go version
 
 # Install SAPMachine
-RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
-    && case "${dpkgArch##*-}" in \
+RUN sapmachine_install() { \
+    SAPMACHINE_MAJOR_VERSION=$(echo ${SAPMACHINE_VERSION} | cut -d. -f1); \
+    ARCH=; \
+    dpkgArch="$(dpkg --print-architecture)"; \
+    case "${dpkgArch##*-}" in \
       amd64) ARCH='amd64';; \
       *) echo "unsupported architecture"; exit 1 ;; \
-    esac \
+    esac; \
+    apt-get update; \
+    apt-get install -y ca-certificates gnupg dirmngr --no-install-recommends; \
+    rm -rf /var/lib/apt/lists/*; \
+    export GNUPGHOME="$(mktemp -d)"; \
+    for key in \
+      CACB9FE09150307D1D22D82962754C3B3ABCFE23 \
+    ; do \
+      gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/sapmachine.gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys "$key" || \
+      gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/sapmachine.gpg --batch --keyserver hkps://keyserver.ubuntu.com --recv-keys "$key" ; \
+    done; \
+    chmod 644 /etc/apt/trusted.gpg.d/sapmachine.gpg; \
+    echo "deb http://dist.sapmachine.io/debian/${ARCH}/ ./" | tee /etc/apt/sources.list.d/sapmachine.list; \
+    apt-get update; \
+    apt-get install -y sapmachine-${SAPMACHINE_MAJOR_VERSION}-jdk=${SAPMACHINE_VERSION} --no-install-recommends; \
+    rm -rf "$GNUPGHOME" /var/lib/apt/lists/*; \
+    apt-get remove --purge --autoremove -y ca-certificates gnupg dirmngr; \
+    ln -s /usr/lib/jvm/sapmachine-${SAPMACHINE_MAJOR_VERSION} ${JAVA_HOME}; \
+  }; \
+  sapjvm_install() { \
+    SAPMACHINE_MAJOR_VERSION=$(echo ${SAPMACHINE_VERSION} | cut -d. -f1); \
+    ARCH=; \
+    dpkgArch="$(dpkg --print-architecture)"; \
+    case "${dpkgArch##*-}" in \
+      amd64) ARCH='x64';; \
+      ppc64el) ARCH='ppc64le';; \
+      *) echo "unsupported architecture"; exit 1 ;; \
+    esac; \
+    apt-get update; \
+    apt-get install -y ca-certificates curl libarchive-tools --no-install-recommends; \
+    rm -rf /var/lib/apt/lists/*; \
+    curl -fsSLO --compressed -b 'eula_3_1_agreed=tools.hana.ondemand.com/developer-license-3_1.txt' https://tools.hana.ondemand.com/additional/sapjvm-${SAPMACHINE_VERSION}-${GOOS}-${ARCH}.zip; \
+    # Notice to modify, download the sha1 value auto
+    echo "583dedfeee0c119839a610ddc8c6768d0c044429  sapjvm-${SAPMACHINE_VERSION}-${GOOS}-${ARCH}.zip" | sha1sum -c -; \
+    bsdtar -xvf sapjvm-${SAPMACHINE_VERSION}-${GOOS}-${ARCH}.zip -C /usr/local; \
+    chown root:root /usr/local/sapjvm_${SAPMACHINE_MAJOR_VERSION}; \
+    rm -f sapjvm-${SAPMACHINE_VERSION}-${GOOS}-${ARCH}.zip; \
```

Comment:
Still mixing apples and tomatoes in the env variable namespace ...

Reply:
yes

Reply:
fix it in https://github.com/SAP/cloud-mta-build-tool/commit/b3ea0b94387d71af5ac4e4952a133f0a78328619